gdal/doc/gdal_virtual_file_systems.dox

```diff
@@ -420,6 +420,10 @@ Pays buckets</a>.
 
 The AWS_S3_ENDPOINT configuration option defaults to s3.amazonaws.com. 
 
+The AWS_VIRTUAL_HOSTING configuration option defaults to TRUE. 
+This allows you to configure the two ways to access the buckets, 
```

Comment:
can you explain more the behaviour when the option is set to TRUE or FALSE ?

Reply:
Is this explanation okay? I send another pull request?

The AWS_VIRTUAL_HOSTING configuration option defaults to TRUE. 
This allows you to configure the two ways to access the buckets, see <a href="http://docs.ceph.com/docs/luminous/radosgw/s3/commons/#bucket-and-host-name">Bucket and Host Name</a> for more details.
<ul>
<li>TRUE value, identifies the bucket via a virtual bucket host name, e.g.: mybucket.cname.domain.com
</li>
<li>FALSE value, identifies the bucket as the top-level directory in the URI, e.g.: cname.domain.com/mybucket 
</li>
</ul>